libs/datamodel/core/tests/common.rs

```diff
@@ -271,6 +272,16 @@ impl ErrorAsserts for ErrorCollection {
         self
     }
 
+    fn assert_is_message(&self, msg: &str) -> &Self {
+        if self.errors.len() == 1 {
+            assert_eq!(self.errors[0].description(), msg);
+        } else {
+            panic!("Expected exactly one validation error. Errors are: {:?}", &self);
+        }
```

Comment:
This can be written more briefly as:

```rust
assert_eq!(self.errors.len(), 1, format!("Expected exactly one validation error. Errors are: {:?}", &self));
assert_eq!(self.errors[0].description(), msg);